templates/admin/includes/preview_unpublished_button.html

```diff
@@ -0,0 +1 @@
+<a class="button" href="/{{ url_name }}/{{ original.id }}{{ link_hash }}" target="_blank">{{ button_name }}</a>
```

Comment:
Формируй ссылку в миксине!
```suggestion
<a class="button" href="{{ link }}" target="_blank">{{ button_name }}</a>
```